README.md

```diff
@@ -74,18 +72,15 @@ The Node API methods have the same signature as the API documentation, featuring
 
 The API key is no longer explicitly required on any Node API method invocations. Unless you have disabled the API key when running ZAP, simply provide it on Node API instantiation as mentioned in the [Usage](#usage) section and it will be provided automatically with each request to the ZAP API.
 
-**Callback mode**: If you provide a callback as the last parameter, the callback will be called with error and response arguments, with the response being an object that corresponds to the JSON output of the API call.  
-**Promise mode**: If you wish to use the modern approach, simply don't provide a callback, and a native promise will be returned for you to deal with as you wish.
-
 ## Getting Help
 
 For help using the OWASP ZAP API refer to:
 
-  * [API Documentation](https://www.zaproxy.org/docs/api/);
-  * [OWASP ZAP User Group](https://groups.google.com/group/zaproxy-users) - for asking questions;
+- [API Documentation](https://www.zaproxy.org/docs/api/);
+- [OWASP ZAP User Group](https://groups.google.com/group/zaproxy-users) - for asking questions;
+
+For specific help with this Node API, contact [@njmulsqb](https://github.com/njmulsqb) (the maintainer).
```

Comment:
Better link to the user group.

Reply:
You mean I remove the maintainer line? I just updated the previous one

Reply:
I mean link to the user group instead of a personal account.

Reply:
I see now there's already the link to the group so yeah, it was to remove the line, sorry.